velox/connectors/hive/storage_adapters/gcs/examples/GCSFileSystemExample.cpp

```diff
@@ -24,10 +24,19 @@
 #include <iostream>
 
 DEFINE_string(gcs_path, "", "Path of GCS bucket");
+DEFINE_string(gcs_max_retry_count, "", "Max retry count");
+DEFINE_string(gcs_max_retry_time, "", "Max retry time (seconds)");
```

Comment:
Can be any valid duration string. Let's remove seconds.

Reply:
Fixed!